images/images.qrc

```diff
@@ -979,6 +979,7 @@
         <file>themes/default/mIconFieldGeometry.svg</file>
         <file>themes/default/algorithms/mAlgorithmRectanglesOvalsDiamonds.svg</file>
         <file>themes/default/algorithms/mAlgorithmOffsetLines.svg</file>
+        <file>composer/refreshing_item.svg</file>
```

Comment:
Reminds me, we should move the composer images into themes/default/layouts/ one day (QGIS 4.0 maybe)

Reply:
it's a hidden easter egg for old-school users :upside_down_face:

Reply:
@nyalldawson , heh.